content/news/developer-guide-for-jakarta-ee-10-and-microprofile-6.0/_index.md

```diff
@@ -0,0 +1,785 @@
+---
+title: Developer Guide for Jakarta EE 10 and MicroProfile 6.0
+date: 2023-05-17
+publishDate: 2023-05-17
+type: announcements
+news/tags:
+  - Jakarta EE
+  - MicroProfile
+authors: [{ gh_handle: Emily-Jiang, name: Emily Jiang }]
```

Comment:
Can you link to my eclipse account https://projects.eclipse.org/user/8461 instead?

Reply:
Certainly. I'll update the PR some time today.